runtime/gc_modron_standard/StandardAccessBarrier.cpp

```diff
@@ -92,13 +104,122 @@ MM_StandardAccessBarrier::tearDown(MM_EnvironmentBase *env)
 	MM_ObjectAccessBarrier::tearDown(env);
 }
 
+/**
+ * Unmarked, heap reference, about to be deleted (or overwritten), while marking
+ * is in progress is to be remembered for later marking and scanning.
+ */
+void
+MM_StandardAccessBarrier::rememberObjectToRescan(MM_EnvironmentBase *env, J9Object *object)
+{
+	//TODO SATB re-enable opt?
+	//if (_markingScheme->markObject(env, object, true)) {
```

Comment:
this should be in. otherwise, it will create way too many unnecessary/duplicate remembered objects (and overstress marking work stack)

Reply:
fixed